datasets/compguesswhat/compguesswhat.py

```diff
@@ -0,0 +1,224 @@
+from __future__ import absolute_import, division, print_function
+
+import gzip
+import json
+import os
+
+import nlp
+
+
+class CompguesswhatConfig(nlp.BuilderConfig):
+    """ BuilderConfig for Discofuse"""
+
+    def __init__(self, data_url, splits, gameplay_scenario, **kwargs):
+        """
+
+        Args:
+            balanced: to specify if we want to load the balanced file or the full file
```

Comment:
I don't see `balanced` used but maybe you can explain `gameplay_scenario` in the docstring?